### PR TITLE
reference: change "region miss-peer" to "region check miss-peer"

### PR DESCRIPTION
**Pull request by dcalvin**
### What is changed, added or deleted? 

Updated "region miss-peer" to "region check miss-peer"

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1824

### Which version does your change affect? 

TIDB 2.1/3.0/Dev

### CODE REVIEW

**Comment by lilin90** [comment]
@TaoZhengCN PTAL

**Comment by TaoZhengCN** [comment]
> @TaoZhengCN PTAL

LGTM

File: v2.1/reference/tools/pd-control.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ As a command line tool of PD, PD Control obtains the state information of the cl
 Single-command mode:
 
 ```bash
-./pd-ctl store -d -u http://127.0.0.1:2379
+./pd-ctl store -u http://127.0.0.1:2379
```
Comment (lilin90) [inline v2.1/reference/tools/pd-control.md]: Why change this line? See https://github.com/pingcap/docs-cn/pull/1818#issuecomment-527870946

Comment (lilin90) [inline v2.1/reference/tools/pd-control.md]: Why change this line? See https://github.com/pingcap/docs-cn/pull/1818#issuecomment-527870946

**Comment by dcalvin** [comment]
Closing this PR for the change is already in docs. @TaoZhengCN FYI

**Pull request closed without merging**